build.gradle

```diff
@@ -416,6 +417,24 @@ allprojects {
       }
     }
 
+    ideProbe = { ->
+      repositories {
+        maven { url 'https://packages.jetbrains.team/maven/p/ij/intellij-dependencies' }
+      }
+
+      dependencies {
+        uiTestImplementation project(':testCommon').sourceSets.test.output
+        uiTestImplementation group: 'org.virtuslab.ideprobe', name: "junit-driver_$scalaMajorVersion", version: ideProbeVersion
+        uiTestImplementation(group: 'org.virtuslab.ideprobe', name: "robot-driver_$scalaMajorVersion", version: ideProbeVersion) {
+          // TODO (VirtusLab/ide-probe#162): use a version of ide-probe that doesn't depend on alredy non-existent remote-fixtures v1.1.18
```

Comment:
Basically, JetBrains pulled off a `leftpad`, albeit on a smaller scale... see the referenced issue